packages/valory/protocols/register/README.md

```diff
@@ -0,0 +1,41 @@
+# Register Protocol
+
+## Description
+
+This is a protocol for communication between two AEAs for registration.
+
+## Specification
+
+```yaml
+---
+name: register
+author: valory
+version: 0.1.0
+description: A protocol for communication between two AEAs for registration.
+license: Apache-2.0
+aea_version: '>=1.0.0, <2.0.0'
+protocol_specification_id: valory/register:0.1.0
+speech_acts:
+  register:
+    info: pt:dict[pt:str, pt:str]
+  success:
+    info: pt:dict[pt:str, pt:str]
+  error:
+    error_code: pt:int
+    error_msg: pt:str
+    info: pt:dict[pt:str, pt:str]
+...
+---
+initiation: [register]
+reply:
+  register: [success, error]
+  success: []
+  error: []
+termination: [success, error]
```

Comment:
This is probably not sufficient for our use case. Try think through what the requirements are and then define them here as a specification.